107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -126,6 +111,30 @@ public CacheManager getCacheManager(URI uri, ClassLoader classLoader, Properties
     return cacheManager;
   }
 
+  public Eh107CacheManager createCacheManager(Configuration config) {
```

Comment:
I would drop this method and force the user to give us a URI that we check is unique.